transit/models/base.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime as py_datetime
+from django.db import models
+
+
+class BaseModel(models.Model):
+    last_modified_date = models.DateTimeField(
+        db_column='LastModifiedDate', default=py_datetime.now
```

Comment:
It's good that we aready created some generic model from which other entities derives.
Why did you import datetime as py_datetime?

Reply:
I can change this to use 'datetime' as it originally is. I did this because of habbit from previous projects.

Reply:
Let's keep default datetime, this naming could be confusing for future contributors.

Reply:
yes. Forgot to push this. Let me do this ASAP.